openfisca_france/model/prestations/anah/eligibilite_anah.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from numpy import (select,where)
+from openfisca_france.model.base import *  # noqa analysis:ignore
+
+class rfr_foyer_fiscal(Variable):
+    column = FloatCol
+    entity = Individu
+    label = u"RFR de chaque foyer fiscal distinct, identifié par son déclarant principal"
+    definition_period = YEAR
+
+    def function(individu, period):
+        return individu.has_role(FoyerFiscal.DECLARANT_PRINCIPAL) * individu.foyer_fiscal('rfr', period)
+
+class eligibilite_anah(Variable):
+    column = EnumCol(
+        enum = Enum([
+            u"A vérifier",
+            u"Modestes",
+            u"Très modestes"
+        ])
+    )
+    entity = Menage
+    label = u"Barème d'éligibilité aux aides ANAH"
+    definition_period = YEAR
+
+    def function(menage, period):
+        departement = menage('depcom',period.first_month).astype(int) / 1000
+        in_idf = (departement == 75) + (departement == 77) + \
+                 (departement == 78) + (departement == 91) + \
+                 (departement == 92) + (departement == 93) + \
+                 (departement == 94) + (departement == 95)
+
+        nb_members = menage.nb_persons()
+
+        rfr_foyer_fiscal = menage.members('rfr_foyer_fiscal', period.n_2)
+        rfr = menage.sum(rfr_foyer_fiscal)
+
+        bareme_idf = select(
+            [nb_members==1,nb_members==2,nb_members==3,nb_members==4,nb_members>=5],
+            [select([rfr <= 19875,rfr <= 24194, rfr>0],[0,1,2]),
+             select([rfr <= 29171,rfr <= 35510, rfr>0],[0,1,2]),
+             select([rfr <= 35032,rfr <= 42648, rfr>0],[0,1,2]),
+             select([rfr <= 40905,rfr <= 49799, rfr>0],[0,1,2]),
+             select([rfr <= 46798+((nb_members-5)*5882),rfr <= 56970+((nb_members-5)*7162), rfr>0],[2,1,0])])
+
+        bareme_out = select(
+            [nb_members==1,nb_members==2,nb_members==3,nb_members==4,nb_members>=5],
+            [select([rfr <= 14360,rfr <= 18409, rfr>0],[0,1,2]),
```

Comment:
Si `2` correspond à très modeste, ça ne colle pas (et c'est sans doute ce qui empêche les tests de passer).

Ici on regarde d'abord si le rfr est inférieur à 14360. Si oui on retourne 0.
Sinon, on regarde si le rfr est inférieur à 18409. Si oui on retourne 1.
Sinon, on regarde si le rfr est supérieur à 0 (toujours vrai), et on retourne 2 .

Voir la [documentation de select](https://doc.openfisca.fr/coding-the-legislation/30_case_disjunction.html#multiples-conditions).

A minima, on devrait remplacer  `[0, 1 ,2]` par `[2, 1, 0]`, et de préférence clarifier le code en:

```py
[select([rfr <= 14360, rfr <= 18409],[2, 1], 0),
```
ou
```py
[select([rfr <= 14360, rfr <= 18409, rfr > 18409],[2, 1, 0]),
```